src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -539,38 +541,41 @@ Public Module VerificationHelpers
         End Function
     End Class
 
-    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As StringBuilder)
+    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As PooledStringBuilder)
```

Comment:
> Also it currently only call from methods within this file.

Still, it is better to keep it more general rather than specific to PooledStringBuilder.